### ISSUE TITLE
'window' is not defined in node.js

### ISSUE THREAD

**Issue by maierfelix** [frenchtoast747/webgl-obj-loader#63]
I'm using `.mjs` in my node.js project and I import the loader with:

````js
import OBJ from "webgl-obj-loader";
````

But when importing I get the error that `window` is undefined in `webgl-obj-loader.min.js`. I fixed it by replacing `window` with: `typeof window !== "undefined" ? window : global`.

I'm not sure if that is the right fix to use this project in node, I can create a PR if it's fine

**Comment by frenchtoast747** [comment]
Thanks for pointing this out!I'll take a look and try to get a fix in ASAP!On Aug 18, 2019 9:13 AM, Felix Maier <notifications@github.com> wrote:I'm using .mjs in my node.js project and I import the loader with:
import OBJ from "webgl-obj-loader";
But when importing I get the error that window is undefined in webgl-obj-loader.min.js. I fixed it by replacing window with: typeof window !== "undefined" ? window : global.
I'm not sure if that is the right fix to use this project in node, I can create a PR if it's fine

**Comment by maierfelix** [comment]
ping, hello world

**Comment by frenchtoast747** [comment]
I'm so sorry, I completely forgot about this issue. I'm looking at it right now!

**Comment by frenchtoast747** [comment]
Apparently it was a problem with Webpack's `umd` output. This was fixed in 83ce2d5e0affceab31997a4bea9d8d8eb8da8bef by adding
```
output: {
// ...
globalObject: "typeof self !== 'undefined' ? self : this",
// ...
}
```
In the webpack config.
